src/util/sandbox/fd_sandbox_private.h

```diff
@@ -0,0 +1,203 @@
+#ifndef HEADER_fd_src_util_sandbox_fd_sandbox_private_h
+#define HEADER_fd_src_util_sandbox_fd_sandbox_private_h
+
+#include "fd_sandbox.h"
+
+FD_PROTOTYPES_BEGIN
+
+/* Clears the environment variables of the process by explicitly zeroing
+   any memory containing environment variable strings (both keys and
+   values).  The envirnment is also cleared using clearenv(3) to make
+   sure any callers of getenv(3) will no longer see the environment as
+   being present (albeit zeroed).
+
+   Environment variables are a common source of information leakage:
+   they can appear in logs and crash dumps, and are accessible to other
+   processses on the machine.
+   
+   Firedancer does not use environment variables to pass information,
+   but it is possible an operator has passed unneeded variables in when
+   launching the program, and we discard them here to make sure they do
+   not get leaked. */
+   
+void FD_FN_SENSITIVE
+fd_sandbox_private_explicit_clear_environment_variables( void );
+
+/* Checks that the open file descriptors in the current process exactly
+   matches the list of allowed file descriptors provided.
+
+   If a file descriptor is open in the process that is not in the list,
+   or a file descriptor that's present in the list is not open in the
+   process an error is raised and the program is aborted.
+
+   The security of the sandbox depends on the file descriptor table
+   being exactly as expected.  For example if the process has a handle
+   open to /etc/shadow, the sandbox is not secure.
+
+   This is a defense in depth measure and mostly catches programming
+   errors introduced when changing the startup sequence of the program,
+   but it also catches cases where systemd or some other init system has
+   spawned the Firedancer process with unexpected file descriptors still
+   open.
+
+   allowed_file_descriptors_cnt must be less than 256 or the process
+   will be exited with an error. */
```

Comment:
Can you add to the comment that `allowed_file_descriptor` cannot have any duplicates?